apps/studio/src/components/PageEditor/ComponentSelector.tsx

```diff
@@ -0,0 +1,246 @@
+          icon={<BiX />}
+          onClick={onClose}
+        />
+      </Flex>
+      <VStack p="2rem" w="full" gap="1.25rem" alignItems={'start'}>
+        <Section>
+          <SectionTitle title="Basic Building Blocks" />
+          <BlockList>
+            <BlockItem
+              label="Paragraph"
+              icon={BiText}
+              onProceed={onProceed}
+              sectionType="paragraph"
+              description="Add text to your page - lists, headings, paragraph, and links."
+            ></BlockItem>
+            <BlockItem
+              label="Image"
+              icon={BiImage}
+              onProceed={onProceed}
+              sectionType="image"
+              description="TODO"
+            />
+          </BlockList>
+        </Section>
+        <Section>
+          <SectionTitle title="Highlight Important Information" />
+          <BlockList>
+            <BlockItem
+              label="Statistics"
+              icon={BiDollar}
+              onProceed={onProceed}
+              sectionType="statistics"
+              description="TODO"
+            />
+            <BlockItem
+              label="Callout"
+              icon={BiSolidQuoteAltLeft}
+              onProceed={onProceed}
+              sectionType="callout"
+              description="TODO"
+            />
+            <BlockItem
+              label="Text with button"
+              icon={BiSolidHandUp}
+              onProceed={onProceed}
+              sectionType="textWithButton"
+              description="TODO"
+            />
+            <BlockItem
+              label="Text with image"
+              icon={BiImages}
+              onProceed={onProceed}
+              sectionType="textWithImage"
+              description="TODO"
+            />
+          </BlockList>
+        </Section>
+        <Section>
+          <SectionTitle title="Organise Content" />
+          <BlockList>
+            <BlockItem
+              label="Cards"
+              icon={BiCard}
+              onProceed={onProceed}
+              sectionType="cards"
+              description="TODO"
+            />
+            <BlockItem
+              label="Columns"
+              icon={BiColumns}
+              onProceed={onProceed}
+              sectionType="columns"
+              description="TODO"
+            />
+            <BlockItem
+              label="Accordion"
+              icon={BiExpandVertical}
+              onProceed={onProceed}
+              sectionType="accordion"
+              description="TODO"
+            />
+            <BlockItem
+              label="Divider"
+              icon={BiRuler}
+              onProceed={onProceed}
+              sectionType="divider"
+              description="TODO"
+            />
+          </BlockList>
+        </Section>
+        <Section>
+          <SectionTitle title="External Content" />
+          <BlockList>
+            <BlockItem
+              label="Youtube"
```

Comment:
nit: YouTube

Reply:
changed in https://github.com/opengovsg/isomer/pull/187/commits/71703b3913474b61c929c028c71dea8f069681c4